hknweb/academics/models/base_models.py

```diff
@@ -1,5 +1,30 @@
 from django.db import models
+from django.db.models import Count
+
 
 class AcademicEntity(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+
+    """
+    Merges two Academic entities of the same type.
+    to_keep: the entity which should subsume the other one (take on all its children), and whose parent should remain
+    to_remove: the entity which should be removed 
+    """
+    @staticmethod
+    def merge(to_keep, to_remove):

```

Comment:
If `to_remove` is a list of entities to remove, let's specify that using a more descriptive name e.g. "entities_to_remove", by using [typing](https://docs.python.org/3/library/typing.html), or by specifying it properly in the documentation?